hooks/nuke_quickdailies_quicktime_settings.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sys
+
+from tank import Hook
+import nuke
+
+class NukeQuickdailiesGetQuicktimeSettings(Hook):
+    """
+    Allows modifying default settings for Quicktime generation.
+    """
+    def execute(self, **kwargs):
```

Comment:
I think you should pass the nuke node name in here - this would allow for a studio to write code where they can resolve the node object in nuke and perform some custom logic.

Also, i think we should pass in the name of the profile used as a parameter. The use case here being that you might want to have different quicktime settings for different profiles. Or is there another way to achieve that? Seems like a common use case we should anticipate.